dataherald/repositories/database_connections.py

```diff
@@ -5,6 +5,10 @@
 DB_COLLECTION = "database_connections"
 
 
+class DatabaseConnectionNotFoundError(Exception):
```

Comment:
Maybe all the Exceptions could have the `Exception` suffix. Like this `DatabaseConnectionNotFoundException`